container.go

```diff
@@ -275,6 +279,10 @@ func (c *ContainerRequest) ShouldBuildImage() bool {
 	return c.FromDockerfile.Context != "" || c.FromDockerfile.ContextArchive != nil
 }
 
+func (c *ContainerRequest) ShouldKeepImage() bool {
+	return c.KeepImage
+}
+
 func (c *ContainerRequest) ShouldPrintBuildLog() bool {
 	return c.FromDockerfile.PrintBuildLog
 }
```

Comment:
This very same pattern makes me confirm that we need to move the `KeepImage` field to `FromDockerfile`, as we are shadowing what we already do for fields in the latter struct

Reply:
:+1: